docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,97 @@
+# Adaptive concurrency limits
```

Comment:
Is this what we want to call the doc? @BC-ACherednichenko @piotrekkaminski

Reply:
We want to avoid any details on the limiting strategy that we use, so I would not use `Adaptive concurrency limits` term at all

I think it is better to have "API rate limiting"